spec/presenters/summary/sections/client_details_spec.rb

```diff
@@ -166,6 +170,22 @@
         expect(answers.count).to eq(7)
       end
     end
+
+    context 'when application is a appeal no changes application' do
+      let(:appeal_no_changes?) { true }
+
+      it 'has the correct rows' do
+        expect(answers.count).to eq(7)
```

Comment:
Is it worth asserting that specific questions/answers are present in this case? I think that would make for a more robust test than just relying on the count.

Reply:
That's a good point, I've added checks to the question labels to assert the expected questions are shown